sdk/proto/go/provider.pb.go

```diff
@@ -2171,6 +2172,8 @@ type ResourceProviderClient interface {
 	Cancel(ctx context.Context, in *empty.Empty, opts ...grpc.CallOption) (*empty.Empty, error)
 	// GetPluginInfo returns generic information about this plugin, like its version.
 	GetPluginInfo(ctx context.Context, in *empty.Empty, opts ...grpc.CallOption) (*PluginInfo, error)
+	// Attach sends the engine address to an already running plugin.
+	Attach(ctx context.Context, in *PluginAttach, opts ...grpc.CallOption) (*empty.Empty, error)
```

Comment:
This is going to break implementers of `ResourceProviderClient`. Which if we're going to have to push out a break change we may as well change NewProviderServer to take a GrpcProvider as well.